.circleci/config.yml

```diff
@@ -90,3 +117,44 @@ jobs:
             mkdir -p $HOME/testground/plans
             ln -s $HOME/project/plans/placebo $HOME/testground/plans/placebo
             go test -v ./...
+
+  # Tests the k8s runner can successfully submit jobs.
+  kind-test:
+    executor: linux
+    steps:
+      - setup-install-bins
+      - run:
+          name: "build testground"
+          command: make install
+      - run:
+          name: "configure cluster"
+          command: |
+            kind create cluster --wait 90s
+            kubectl apply -f .circleci/pv.yaml
+            kubectl apply -f .circleci/pvc.yaml
+            kubectl label nodes kind-control-plane testground.nodetype=plan
+      # The placebo does not require a sidecar.
+      # To prevent kind from attempting to download the image from docker hub,
+      # build and load the image before executing it.
+      # The plan is renamed as `testplan:placebo` because kind will check dockerhub if the tag is `latest`.
+      - run:
+          name: "run a plan"
+          command: |
+            testground daemon &
+            testground plan import --from plans/placebo
+            testground build single --builder docker:go --plan placebo | tee build.out
+            export ARTIFACT=$(awk -F\" '/generated build artifact/ {print $8}' build.out)
+            docker tag $ARTIFACT testplan:placebo
+            kind load docker-image testplan:placebo
+            testground run single --runner cluster:k8s --builder docker:go --use-build testplan:placebo --instances 1 --plan placebo --testcase ok | tee run.out
+      - run:
+          name: "verify output"
```

Comment:
I suggest we join the `run a plan` and `verify output` into one CircleCI `run`, and extract in `sh` that we can modify and add more tests to, rather than have to edit the circleci.yml to add more testplan runs.

I imagine we want to run testplans serially on the `testground daemon` for now.

WDYT?